src/utils/Decimals.ts

```diff
@@ -0,0 +1,5 @@
+import Decimal from 'decimal.js'
+export default function reduceDecimals(number: string) {
```

Comment:
Please add an extra param which defaults to 18.
It's going to be super useful in the future, when dealing with tokens that do not use 18 decimals